governance/votes/Executive vote - January 24, 2024.md

```diff
@@ -0,0 +1,143 @@
+into USDC and transfer the USDC to the Urn Input Conduit deployed at [0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1](https://etherscan.io/address/0x08012ec53a7fabf6f33318dfb93c1289886ebbe1).
+
+To execute this process, the following action will be taken if the executive proposal passes:
+
+- The DAO Resolution with IPFS hash [QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w](https://ipfs.io/ipfs/QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w) will be approved.
+- `RWA009_A_INPUT_CONDUIT_URN_USDC` deployed at `0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1` will be added to the chainlog on Mainnet.
+
+### AVC Members Compensation Q4 2023
+
+ **Authorization:** [Atlas 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)\
+ **Proposal:** [Forum post "AVC Member Participation Rewards - Q4 2023"](https://forum.makerdao.com/t/avc-member-participation-rewards-q4-2023/23458)
+
+ If this executive proposal passes, the following distributions will be made for Q4 AVC member compensation, totaling **208.5 MKR**:
+
+| Delegate | Address | Amount (MKR) |
+|--|--|--|
+| IamMeeoh | [0x47f7A5d8D27f259582097E1eE59a07a816982AE9](https://etherscan.io/address/0x47f7A5d8D27f259582097E1eE59a07a816982AE9) | 20.85 MKR |
+| DAI-Vinci | [0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a](https://etherscan.io/address/0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a) | 20.85 MKR |
+| opensky | [0xf44f97f4113759E0a57756bE49C0655d490Cf19F](https://etherscan.io/address/0xf44f97f4113759E0a57756bE49C0655d490Cf19F) | 20.85 MKR |
+| ACRE DAOs | [0xBF9226345F601150F64Ea4fEaAE7E40530763cbd](https://etherscan.io/address/0xBF9226345F601150F64Ea4fEaAE7E40530763cbd) | 20.85 MKR |
+| fhomoney.eth | [0xdbD5651F71ce83d1f0eD275aC456241890a53C74](https://etherscan.io/address/0xdbD5651F71ce83d1f0eD275aC456241890a53C74) | 20.85 MKR |
+| Res | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | 20.85 MKR |
+| Harmony | [0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3](https://etherscan.io/address/0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3) | 20.85 MKR |
+| Libertas | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 20.85 MKR |
+| seedlatam.eth | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d) | 20.85 MKR |
+| 0xRoot | [0xC74392777443a11Dc26Ce8A3D934370514F38A91](https://etherscan.io/address/0xC74392777443a11Dc26Ce8A3D934370514F38A91) | 20.85 MKR |
+
+
+### Trigger Spark Proxy Spell
```

Comment:
Is this change being included in the proxy spell?

https://vote.makerdao.com/polling/QmeWioX1

Reply:
You're right of course. Apologies.